packages/@lightningjs/ui-components/src/components/MetadataBase/MetadataBase.d.ts

```diff
@@ -67,11 +67,6 @@ declare namespace MetadataBase {
      * sets the marquee for Title and Description to the same value so they sync
      */
     marquee?: boolean;
-    /**
-     * @deprecated
-     * relevant content data in the middle
-     */
-    subtitle?: string;
```

Comment:
Was there a reason to not add the subtitle back into MetadataBase and only into MetadataCardContent?

If we're keeping it the way you have it now, we might want @anthony9187's thoughts on how to properly label `subtitle` being deprecated on MetadataBase but being added to MetadataCardContent.

Reply:
Hey, Nothing specific but MetadataCardContent is more appropriate for flex use case.

Reply:
Can we move this and have `subtitle` added back to MetadataBase, implement it in MetadataCardContent, and leave it available but unused in the rest of the Metadata components?

Reply:
I feel like it would be confusing to have it available but unused in other Metadata variants

Reply:
Sorry, when I said "unused", I meant that we would just not update the stories to use the new property since we don't have designs for it. But if users want it, the components would have the property and would have it implemented properly.

I'm fine with leaving it as is since this is the only case (so far) of the subtitle being needed.

Reply:
ahhh gotcha. yeah if the code supports it then im all for it